docs/tutorials/google/start.ipynb

```diff
@@ -1,478 +1,532 @@
 {
```

Comment:
<p>Update "best practices" link to <a href="https://quantumai.google/cirq/google/best_practices" target="_blank">https://quantumai.google/cirq/google/best_practices</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>